projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-card-hero-img',
+  templateUrl: './card-hero-img.component.html',
+  styleUrls: [ './card-hero-img.component.scss' ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgCardHeroImageComponent {
+  @Input() coverImageUrl: string;
+  @Input() imageSrc: string;
+  @Input() imageAlt = 'card-hero-img';
+  @HostBinding('class') get class(): string {
+    if (this.coverImageUrl || this.imageSrc) {
+      return 'lg-card-hero-img';
```

Comment:
I think using bem the classes should be `lg-card-hero__img` and `lg-card-hero__icon`

Reply:
🤔 Done :)